docs/index.html

```diff
@@ -0,0 +1,445 @@
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <div class="tabs-content" id="blog">
+        <div class="container">
+            <div class="row">
+                <div class="wrapper">
+                    <div class="col-md-4">
+                        <div class="section-heading">
+                            <h4>Support Documentation</h4>
+                            <div class="line-dec"></div>
+                            <p>Find modules here, for reference and a quick start.</p>
+                            <ul class="tabs clearfix" data-tabgroup="first-tab-group">
+                              <li><a href="#tab1" class="active">Service Rest</a></li>
+                                <li><a href="#tab2">Client Rest</a></li>
+                                <li><a href="#tab3">Couchbase</a></li>
+                                <li><a href="#tab4">Synapse Module Architecture&nbsp;</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <section id="first-tab-group" class="tabgroup">
+                            <div id="tab1">
+                                <img src="img/blog_item_01.jpg" alt="">
+                                <div class="text-content">
+                                  <h4>Service Rest</h4>
+                                    <span><a href="#">Synapse</a> / <a href="#">21 September 2021</a></span>
+                                  <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities.</p>
+                                </div>
+                            </div>
+                            <div id="tab2">
+                                <img src="img/blog_item_02.jpg" alt="">
+                                <div class="text-content">
+                                    <h4>Client Rest</h4>
+                                    <span><a href="#">Synapse</a> /  <a href="#">24 August 2020</a></span>
+                                    <p>This is the synapse gateway framework utilized to consume RESTful APIs. It provides several out-of-the-box functionalities</p>
+                                </div>
+                            </div>
+                            <div id="tab3">
+                                <img src="img/blog_item_03.jpg" alt="">
+                                <div class="text-content">
+                                    <h4>Couchbase</h4>
+                                    <span><a href="#">Synapse</a> /  <a href="#">18 July 2020</a></span>
+                                    <p>This is the synapse couchbase abstraction framework used whenever there is a need to connect to couchbase database and read from it. It provides several out-of-the-box functionalities. </p>
```

Comment:
Can you uppercase all the S in Synapse and all the C in Couchbase

Reply:
done